### PR TITLE
removed time-costly validator: if-emoji; removed ~1500 appends, inste…

### PR DESCRIPTION
**Pull request by lukiq**
…d do only 8

### CODE REVIEW

**Review by MichalPaszowski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ This project is licensed under the GPL v3 License - see the [LICENSE.md](LICENSE
 
 ## Changelog
 
+**1.3.5**
```
Comment (quarties) [inline README.md]: should be `1.3.6` - I don't know why `1.3.5` is missing in this file

File: js/client.js [hunk 2]
```diff
@@ -15,13 +14,8 @@ window.EmojiKeyboard = {
...
         const that = this;
         $.each(emojis, function (category) {
-            let elementID = category.toLowerCase(),
-                anyEmojiSupported = 0;
-            const emojiCategoryDOM = $('<p>', {
-                class: 'emoji-category',
-                id: elementID,
-            });
 
+            $emojies = "";
```
Comment (quarties) [inline js/client.js]: Why `$` sign? Also missing `let` declaration before variable and it should be `emojis` not `emojies`.

File: js/client.js [hunk 3]
```diff
@@ -30,20 +24,12 @@ window.EmojiKeyboard = {
...
                 let emoji = String.fromCodePoint.apply(this, dec);
-                if (ifemoji(emoji)) {
-                    emojiCategoryDOM.append($('<span>', {
-                        class: 'emoji-single',
-                        'data-emoji-code': emoji,
-                        html: emoji
-                    })); // '<span class="emoji-single" data-emoji-code="' + emoji + '">' + emoji + '</span>');
-                    anyEmojiSupported++;
-                }
+                $emojies += '<span class="emoji-single" data-emoji-code="' + emoji + '">' + emoji + '</span>';
```
Comment (quarties) [inline js/client.js]: As we use babel for this project I suggest to use es6 syntax:
```javascript
emojis += `<span class="emoji-single" data-emoji-code="${emoji}">${emoji}</span>`;
```

**Review by quarties — CHANGES_REQUESTED** [review]
Please also bump version to 1.3.6 in `package.json` file

**Review by quarties — APPROVED** [review]
Please bump version in `package.json` file (`npm version`)

**Pull request merged**